### DISCUSSION TITLE
Support for rainbow colour text

### DISCUSSION THREAD

**Discussion by tsctrl** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#221] [Q&A]
hi @mrfaptastic,
this might be on the gfx lib changes. not sure but,
is there any one has successfully adds rainbow colorful gradients text as what normally we saw on the chinese led display controller? do you have idea how this can be done?

![images (3)](https://user-images.githubusercontent.com/75555993/146660585-b5ba8e85-b73b-495f-b172-ed488164d07f.jpeg)

thanks!

**Comment by mrcodetastic** [comment]
Yeah, this is a graphics routine/library question I'm afriad.

This library is lower level and just lights up pixels on a panel the way you code it to do so.
